src/wasm.rs

```diff
@@ -51,3 +55,39 @@ impl Encoder {
             .collect()
     }
 }
+
+#[wasm_bindgen]
+pub struct EncodingPacket {
+    source_block_number: u8,
+    encoding_symbol_id: u32,
+    data: Vec<u8>,
+}
+
+#[wasm_bindgen]
+impl EncodingPacket {
+    #[wasm_bindgen]
+    pub fn deserialize(data: &[u8]) -> EncodingPacket {
+        let payload_data = [data[0], data[1], data[2], data[3]];
+        let payload_id = PayloadId::deserialize(&payload_data);
+        EncodingPacket {
+            source_block_number: payload_id.source_block_number(),
+            encoding_symbol_id: payload_id.encoding_symbol_id(),
+            data: Vec::from(&data[4..]),
+        }
+    }
+
+    #[wasm_bindgen]
+    pub fn source_block_number(&self) -> u8 {
+        self.source_block_number
+    }
+
+    #[wasm_bindgen]
+    pub fn encoding_symbol_id(&self) -> u32 {
+        self.encoding_symbol_id
+    }
```

Comment:
Can you make the interface of this match the `EncodingPacket` [exposed in the Rust lib](https://github.com/cberner/raptorq/blob/11d2de97f25508bbc56ce02ac6aebb7c96a59493/src/base.rs#L84)? That is replace `source_block_number()` & `encoding_symbol_id()` with a `payload_id()` method, or alternately rename this struct to `PayloadId` and make it match that struct in the Rust lib.